sources/Renderer/Direct3D11/D3D11RenderSystem.cpp

```diff
@@ -671,9 +692,10 @@ void D3D11RenderSystem::QueryDXDeviceVersion()
 {
     LLGL_ASSERT_PTR(device_);
 
+    HRESULT hr = S_OK;
```

Comment:
Moved this here, because line 681/703 would fail to compile if LLGL_D3D11_ENABLE_FEATURELEVEL is set to 2